src/Microsoft.AspNetCore.Server.IISIntegration.Tools/project.json

```diff
@@ -0,0 +1,39 @@
+{
+  "version": "1.1.0-preview4-final",
```

Comment:
We don't intend to. When I merge back to dev, this source code won't be making the trip back into the dev branch